src/v/cluster/partition_balancer_backend.cc

```diff
@@ -343,6 +346,9 @@ ss::future<> partition_balancer_backend::do_tick() {
       _state.topics().has_updates_in_progress()
       || plan_data.status == planner_status::actions_planned) {
         _cur_term->last_status = partition_balancer_status::in_progress;
+    } else if (plan_data.status == planner_status::missing_sizes) {
```

Comment:
am I missing something, missing_sizes is not assigned anywhere to status? Where is it set?

Reply:
i was blind